dipy/viz/ui.py

```diff
@@ -2952,6 +2952,278 @@ def set_img(self, img):
         self.texture = set_input(self.texture, img)
 
 
+class Option(UI):
+
+    """
+    A set of a Button2D and a TextBlock2D to act as a single option
+    for checkboxes and radio buttons.
+    Clicking the button toggles its checked/unchecked status.
+
+    Attributes
+    ----------
+    label : str
+        The label for the option.
+    font_size : int
+        The size of the font for the label.
+    font_size : int
+            Font Size of the label.
+    """
+
+    def __init__(self, label, position=(0, 0), font_size=18):
+        """
+        Parameters
+        ----------
+        label : str
+            Option's label.
+        position : (float, float)
+            Absolute coordinates (x, y) of the lower-left corner of
+            the button of the option.
+        font_size : int
+            Font Size of the label.
+        """
+        self.label = label
+        self.font_size = font_size
+        self.checked = False
+        self.button_size = (font_size * 1.2, font_size * 1.2)
+        super(Option, self).__init__(position)
+
+    def _setup(self):
+        """ Setup this UI component.
+        """
+        # Option's button
+        self.button_icons = []
+        self.button_icons.append(('unchecked',
+                                 read_viz_icons(fname="stop2.png")))
+        self.button_icons.append(('checked',
+                                 read_viz_icons(fname="checkmark.png")))
+        self.button = Button2D(icon_fnames=self.button_icons,
+                               size=self.button_size)
+
+        self.text = TextBlock2D(text=self.label, font_size=self.font_size)
+
+    def _get_actors(self):
+        """ Get the actors composing this UI component.
+        """
+        return self.button.actors + self.text.actors
+
+    def _add_to_renderer(self, ren):
+        """ Add all subcomponents or VTK props that compose this UI component.
+
+        Parameters
+        ----------
+        ren : renderer
+        """
+        self.button.add_to_renderer(ren)
+        self.text.add_to_renderer(ren)
+
+    def _get_size(self):
+        width = self.button.size[0] + 10 + self.text.size[0]
```

Comment:
10 is a magic number here and in line 3035. It should be pulled out into a variable so the two don't get out of sync.